pkg/errors/sdk_errors.go

```diff
@@ -7,17 +7,15 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"io"
 	"net/http"
 )
 
-const err = "error"
+const errorKey = "error"
 
 var (
-	// ErrJSONErrKey indicates response body did not contain erorr message.
-	errJSONKey = New("response body expected error message json key not found")
-
-	// ErrUnknown indicates that an unknown error was found in the response body.
-	errUnknown = New("unknown error")
+	// Failed to read response body.
+	errRespBody = errors.New("Failed to read response body")
```

Comment:
Error string should not be capitalized or end with punctuation mark

Reply:
@mteodor , Done